src/lighteval/tasks/default_prompts.py

```diff
@@ -196,18 +195,17 @@ def bbh_lighteval(line, task_name: str = None):
         query=query,
         choices=LETTER_INDICES[: len(line["choices"])],
         gold_index=line["target_idx"],
-        target_for_fewshot_sorting=LETTER_INDICES[: len(line["choices"])],
         instruction=line.get("task_prefix", None),
     )
 
 
 def bbh(line, instruction, choices, task_name: str = None):
+    is_few_shots = line.get("__few_shots", False)
     return Doc(
         task_name=task_name,
         query=f"{instruction}Q: {line['input']}\nA:",
-        choices=choices,
+        choices=[(" " if is_few_shots else "") + c for c in choices],
```

Comment:
why did that changed ?

Reply:
Hm indeed it's not actually needed. Will revert